src/portal/src/i18n/lang/fr-fr-lang.json

```diff
@@ -327,6 +333,8 @@
         "DELETED_SUCCESS": "Membre supprimé avec succès.",
         "SWITCHED_SUCCESS": "Rôle du membre changé avec succès.",
         "OF": "sur",
+        "SWITCH_TITLE": "Confirmez le changement de membres projet",
+        "SWITCH_SUMMARY": "Voulez-vous changer les membres projet {{param}}?",        
```

Comment:
What is containing param ?

Reply:
I found the formulation weird

Reply:
This text doesn't seem to be used at all by the application. I can't find it referenced anywhere. So for now, I will leave it as it is.